### ISSUE TITLE
[Docs: 1.2 Quickstart] kubeval should be configured with `strict: true`

### ISSUE THREAD

**Issue by frankfarzan** [GoogleContainerTools/kpt#1862]

**Comment by mengqiy** [comment]
I changed it in https://github.com/GoogleContainerTools/kpt/pull/1797/files. Did it get reverted somehow?

**Comment by phanimarupaka** [comment]
I reverted it as I was facing an issue. 

```
package ".": running function "gcr.io/kpt-fn/kubeval:v0.1": FAILED
fn.render: pkg .:
	pkg.render:
	pipeline.run: [ERROR] Failed to parse raw kubeval output:
Unexpected token E in JSON at position 0

ERR  - stdin: Failed initializing schema file:///tmp/master-standalone-strict/deployment-apps-v1.json: open /tmp/master-standalone-strict/deployment-apps-v1.json: no such file or directory
 in object 'apps/v1/Deployment//my-nginx' in file deployment.yaml
[ERROR] Failed to parse raw kubeval output:
Unexpected token E in JSON at position 0

ERR  - stdin: Failed initializing schema file:///tmp/master-standalone-strict/service-v1.json: open /tmp/master-standalone-strict/service-v1.json: no such file or directory
 in object 'v1/Service//my-nginx-svc' in file svc.yaml
```

**Comment by phanimarupaka** [comment]
But we should not add it back until the issue is addressed as users might not know to run `docker pull`. WDYT?

**Comment by mengqiy** [comment]
SGTM

**Comment by mengqiy** [comment]
Closed by #2029
